docs/other-topics/scopes.md

```diff
@@ -2,9 +2,19 @@
 title: Scopes
 ---
 
-Scopes are used to help you reuse code. You can define commonly used queries, specifying options such as `where`, `include`, `limit`, etc.
+:::warning
+
+Scopes are a fragile feature that we do not recommend using beyond very simple scopes. Proceed with caution.
+
+:::
+
+:::info
 
-This guide concerns model scopes. You might also be interested in the [guide for association scopes](../advanced-association-concepts/association-scopes.md), which are similar but not the same thing.
+This section concerns association scopes, not to be confused with [association scopes](../associations/association-scopes.md).
+
+:::
+
```

Comment:
Can't comment on line 66, needs to go to `pathname:///api/v7/classes/_sequelize_core.model#addScope`